File: test/fixtures/elixir/get_long_form_header.ex
request = %HTTPoison.Request{
  method: :get,
  url: "http://localhost:28139/api/retail/books/list",
  options: [],
  headers: [
    {~s|Accept|, ~s|application/json|},
    {~s|user-token|, ~s|75d7ce4350c7d6239347bf23d3a3e668|},
  ],
  params: [],
  body: ""
}

response = HTTPoison.request(request)

File: test/fixtures/r/get_long_form_header.r
library(httr)

headers = c(
  Accept = "application/json",
  `user-token` = "75d7ce4350c7d6239347bf23d3a3e668"
)

res <- httr::GET(url = "http://localhost:28139/api/retail/books/list", httr::add_headers(.headers=headers))
